package.json

```diff
@@ -29,6 +29,7 @@
     "source-map": "^0.4.4"
   },
   "devDependencies": {
+    "babel-plugin-transform-function-bind": "^6.8.0",
```

Comment:
Ah, right. Durr... There has to be a better way for that though. Doesn't really matter though. /future